docs/EC-UI-screenshot.md

```diff
@@ -0,0 +1,135 @@
+# EC WEB-UI Screenshots as per Tabs
+
+
+## 1)View:
+
+1.1)View: Topology View Table
+
+![Alt text](/docs/Ec%20Screenshot/View/ViewTopology.png?raw=true "Optional Title")
+
+
+1.2)Tree list and Topology View: 
+
+
+![Alt text](/docs/Ec%20Screenshot/View/ViewTreelist.png?raw=true "Optional Title")
+
+
+
+
+
+## 2)Maintain:
+
+2.1)Maintain: Subscription ->Create
```

Comment:
Please use proper markdown indenting.

Reply:
@Subhash-vishwakarma please assist @vidya22022

Reply:
@ayasuda-ge Vidhya raised a new PR using proper markdown indenting. Please have a review and let us know if any further changes required